manual/src/ornate/reference.md

```diff
@@ -68,6 +68,17 @@ useYarn := true
 
 The `yarn` command must be available in the host platform.
 
+### Reload Workflow {#reload-workflow}
+
+To get a faster "change source and reload application" workflow enable the setting `enableReloadWorkflow`.
+
+The reload workflow is a mode that replaces the `webpack` task implementation with a different one,
+that does not use webpack to process the output of the Scala.js compilation. 
```

Comment:
I would also add:

> Instead, it pre-bundles the modules imported by your application and exposes them to the global namespace. Since these dependencies can then be resolved from the global namespace, the output of Scala.js is just concatenated after the contents of the pre-bundling process.